client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/list-item.js

```diff
@@ -1,125 +0,0 @@
-/** @format */
```

Comment:
Moved testing behaviors and expectations to `client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/test/index.js`